admin/app/AppLoader.scala

```diff
@@ -30,7 +30,7 @@ class AppLoader extends FrontendApplicationLoader {
 trait AdminServices {
   def wsClient: WSClient
   def akkaAsync: AkkaAsync
-  lazy val capiHttpClient = wire[CapiHttpClient]
+  private lazy val capiHttpClient = wire[CapiHttpClient]
```

Comment:
Making it private because it is only necessary for initializing the ContentApiClient (one line below) and therefore avoiding conflicting member with different type in preview when using a specific `trainingCapiHttpClient` 

@alexduf let me know if you can think of a more elegant way to solve this